target/linux/ipq40xx/mikrotik/base-files/etc/board.d/02_network

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+#
+# Copyright (c) 2015 The Linux Foundation. All rights reserved.
+# Copyright (c) 2011-2015 OpenWrt.org
+#
+
+. /lib/functions/uci-defaults.sh
+. /lib/functions/system.sh
+
+ipq40xx_setup_interfaces()
+{
+	local board="$1"
+
+	case "$board" in
+	mikrotik,routerboard-wap-g-5hacd2hnd)
+		ucidef_set_interfaces_lan_wan "eth0" "eth1"
+		ucidef_add_switch "switch0" \
+			"0u@eth0" "1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1"
```

Comment:
untagged intended?

Reply:
I removed the switch here. The device has only 2 interfaces. The target defaults work fine for this device.